site/content/docs/integrations/prometheus.md

```diff
@@ -55,6 +78,7 @@ Notice that:
 - The `checkly_check_status` metric has `muted` and `activated` labels, reflecting if a check is sending out alerts or is actually
 running.
 - The `checkly_check_result` metric has a `region` label.
+- The private location "Internal CI" has one Checkly Agent connected. From `checkly_private_location_queue_size` and `checkly_private_location_oldest_scheduled_check_run`, we see that there's no backlog of check run jobs.
 
 
 
```

Comment:
Can we link to the https://app.checklyhq.com/settings/account/integrations page here?
When we tell a user to go somewhere on the WebApp, I'd like to link it. Other than that LGTM.

Reply:
Good idea :) I'll open another PR

Reply:
Done - https://github.com/checkly/checklyhq.com/pull/433